indigo/distribution.yaml

```diff
@@ -4221,6 +4221,12 @@ repositories:
       url: https://github.com/kf/kingfisher.git
       version: indigo-devel
     status: maintained
+  kml_util:
+    doc:
+      type: git
+      url: https://github.com/silviomaeta/kml_util.git
+      version: indigo-devel
```

Comment:
There is no such branch name in your repo. This should probably be `master`?

Same below.